client/shared/src/graphql/persistenceMapper.ts

```diff
@@ -0,0 +1,64 @@
+import { pick } from 'lodash'
+
+/**
+ * Hardcoded names of the queries which will be persisted to the local storage.
+ * After the implementation of the `persistLink` which will support `@persist` directive
+ * hardcoded query names will be deprecated.
+ */
+export const QUERIES_TO_PERSIST = ['viewerSettings', 'extensionRegistry']
```

Comment:
Would be useful to have this typed, this could work?

```suggestion
export const QUERIES_TO_PERSIST: (keyof IQuery)[] = ['viewerSettings', 'extensionRegistry']
```